.github/workflows/linux.yml

```
@@ -43,7 +43,7 @@ jobs:
       - name: Test
         run: bin/ci build
 
-  test_preview_mt:
+  x86_64-gnu-test_preview_mt:
```

I think `preview_mt` should be a separate component (not mingled with `test`).

```suggestion
  x86_64-gnu-test-preview_mt:
```